### PR TITLE
Product category, payment type, quantity, price, failure reason generation

### PR DESCRIPTION
**Pull request by TonyErazo**
Added the generation for the following field names: product category, payment type, quantity, price, failure reason.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,7 +1,46 @@
...
+
+    print('Category: {0}, Payment Type: {1}, Quantity: {2}, Price: {3}, Failure Reason: {4}'.format(product_category, payment_type, quantity, price, failure_reason))
+
+def getProductCategoryProvider():
+    product_category_provider = DynamicProvider(
+        provider_name = 'category',
+        elements = ['Beauty', 'Books', 'Electronics', 'Fine Art', 'Health & Personal Care', 
+                        'Home & Garden', 'Music & DVD', 'Musical Instruments', 'Office Products', 
+                        'Outdoors', 'Pet Supplies', 'Software', 'Sports', 'Toys & Games', 'Video Games', 
+                        'Video, & Blu-ray', 'Watches'],
```
Comment (Rainmaker0102) [inline main.py:27]: Change "Video, & Blu-ray" to "Video & Blu-Ray"

File: main.py [hunk 2]
```diff
@@ -1,7 +1,46 @@
...
+    )
+    return product_category_provider
+
+def getPaymentProvider():
```
Comment (Rainmaker0102) [inline main.py:31]: nit: put function definitions that aren't main should go above main

File: main.py [hunk 3]
```diff
@@ -1,7 +1,46 @@
...
+    fake.add_provider(getFailureProvider())
+
+
+    product_category = fake.category()
+    payment_type = fake.payment_type()
+    quantity = fake.pyint(min_value=1, max_value=20)
+    price = round(fake.pyfloat(min_value=1.00, max_value=200.00), 2)
+    failure_reason = fake.failure_reason()
+
+    print('Category: {0}, Payment Type: {1}, Quantity: {2}, Price: {3}, Failure Reason: {4}'.format(product_category, payment_type, quantity, price, failure_reason))
```
Comment (Rainmaker0102) [inline main.py:19]: An f string would be more readable here.
```
print(f'Category: {product_category}, Payment Type: {payment_type}, Quantity: {quantity}, Price: {price}, Failure Reason: {failure_reason}')
```

**Review by Rainmaker0102 — APPROVED** [review]
Looks good! Typographical and readability recommendations are in the code

**Comment by TonyErazo** [comment]
Making the changes mentioned by Sean

**Pull request closed without merging**